src/nl/hannahsten/texifyidea/ui/SkimConfigureInverseSearchDialog.kt

```diff
@@ -0,0 +1,31 @@
+package nl.hannahsten.texifyidea.ui
+
+import com.intellij.openapi.ui.DialogBuilder
+import java.awt.BorderLayout
+import javax.swing.JLabel
+import javax.swing.JPanel
+
+
+/**
+ * A dialog that contains a brief explanation about configuring inverse search with Skim.
+ *
+ * @author Stephan Sundermann
+ */
+class SkimConfigureInverseSearchDialog {
+    init {
+        DialogBuilder().apply {
+            setTitle("Configure inverse search")
+
+            val body = JLabel("<html>In the Skim settings, go to the Sync tab. Select Custom as Preset, and as a command provide 'idea' and set arguments to '--line %line %file' <br><br>See the wiki for more information.</html>")
```

Comment:
We're discussing making the wiki publicly editable so you can add the documentation yourself (similar to Okular). Will let you know once we have a decision. Preferably that would include a screenshot of this setting, gifs are also possible. (wiki is written in asciidoc)

Reply:
Alright